dev/drivers/scripts/plots/mesoscale/jevs_mesoscale_precip_plots.sh

```diff
@@ -48,8 +48,8 @@ evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
 export PYTHONPATH=$HOMEevs/ush/$COMPONENT:$PYTHONPATH
 
 # Developer Settings
-export COMIN=/lfs/h2/emc/vpppg/noscrub/${USER}/$NET/$evs_ver_2d
```

Comment:
Change COMIN back to `/lfs/h2/emc/vpppg/noscrub/${USER}/$NET/$evs_ver_2d`

Reply:
Was there a commit for this fix? I still see HOMevs set as "/lfs/h2/emc/vpppg/noscrub/${USER}/EVS_mesoscale_v2/EVS"

Reply:
Should be here! https://github.com/NOAA-EMC/EVS/pull/594/commits/ac1eba0c7c31e70749f76dbed07032153e85cc52